src/main/java/com/google/sps/data/Parse.java

```diff
@@ -0,0 +1,71 @@
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.ArrayList;
+import com.google.sps.data.Comment;
+import com.google.sps.data.Constants;
+import com.google.sps.data.Post;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key; 
+import com.google.appengine.api.datastore.KeyFactory;
+
+public class Parse {
```

Comment:
why is this a separate class from InsertAndUpdate?

I think along with the rename comment, these should be in the same class (unless I'm missing something).

Reply:
InsertAndUpdate puts stuff in datastore. Parse takes stuff out of datstore and packages it into java classes.